FajriFadli-Jakarta.md

```diff
@@ -0,0 +1,10 @@
+# Jakarta
+
+### Restaurants
+Fast foods
+
+### Attraction for Tourists
+Traffic jam
```

Comment:
@clangue According to one study in early 2015, it's the most congested city in the world. Late 2015, we're the 11th, so good news I guess.